test/unit/dialects/oracledb.js

```diff
@@ -181,6 +181,41 @@ describe('OracleDb mix of proxy/non-proxy request', function() {
   });
 });
 
+describe('OracleDb request for transactions', function() {
```

Comment:
Je crois que les cas de tests seraient plutôt :
- obtenir la valeur USER sans l'utilisation du proxy user et constater que c'est l'utilisateur utilisé pour se connecter à la BD
- obtenir la valeur USER avec l'utilisation du proxy user et constater que c'est l'utilisateur proxy plutôt que celui utilisé pour se connecter à la base de données

Reply:
J'ai ajouté un autre cas de test: `Generate transaction with proxy user in transactions` Valide que l'utilisateur saisi au niveau du proxy en tant que paramètre d'une transaction correspond à l'utilisateur connecté à la base de données.

Au total, il existe trois cas, pour vérifier la modification apportée à ce PR:
- `Generate transaction`: simplement valide une transaction, pour montrer que la modification n'a pas généré de problèmes à ce niveau.
- `Generate transaction with proxy user in options with fail result`: Un transaction est effectué avec un utilisateur proxy via les .options, ce qui montre que cette modification ne fonctionne pas au niveau de la transaction.
- et le dernier expliqué au début du commentaire